src/components/admin/event/EventDetailsFormItem/style.module.scss

```diff
@@ -0,0 +1,44 @@
+.formItem {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+
+  &[data-error='true'] {
+    input,
+    textarea,
+    select {
+      border: 1px solid #f00;
+    }
+  }
+
+  input,
+  textarea,
+  select {
+    border: 1px solid #bbb;
+    border-radius: 4px;
+    padding: 4px;
+    transition: 0.3s ease-in-out border;
+    width: 100%;
+
+    &:focus {
+      border: 1px solid var(--theme-primary-2);
+    }
+
+    &:hover {
+      border: 1px solid var(--theme-text-on-background-1);
+    }
+  }
+
+  textarea {
+    height: 5rem;
+    resize: vertical;
+  }
+
+  .formError {
+    color: #f00;
```

Comment:
will this show up well in dark mode? red on black

Reply:
I think it looks fine imo:
<img width="1433" alt="Screenshot 2023-10-18 at 11 34 01 PM" src="https://github.com/acmucsd/membership-portal-ui-v2/assets/27360825/be5e7567-d04d-484d-be17-c0adc3b785c0">